docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +112,95 @@ significant changes in each minor release.
-To do this we have a deprecation cycle. First we mark a feature as 'deprecated'. This is normally accompanied with warnings
-to the user as to why we deprecated it, what alternatives they should switch to and when (which version) we are scheduled
-to remove the feature permanently.
+Sometimes we remove a feature, normally in favor of a reimplementation that we hope does a better job. To do this we have a deprecation cycle. First we mark a feature as 'deprecated'. This is normally accompanied with warnings to the user as to why we deprecated it, what alternatives they should switch to and when (which version) we are scheduled to remove the feature permanently.
 
 Ansible deprecation cycle
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Since Ansible is a package of individual collections, the deprecation cycle depends on the collection maintainers. We recommend the collection maintainers deprecate a feature in one Ansible major version and do not remove that feature for one year, or at least until the next major Ansible version. For example, deprecate the feature in 2.10.2, and do not remove the feature until 2.12.0.  Collections should use semantic versioning, such that the major collection version cannot be changed within an Ansible major version. Thus the removal should not happen before the next major Ansible release. This is up to each collection maintainer and cannot be guaranteed.
+Since Ansible is a package of individual collections, the deprecation cycle depends on the collection maintainers. We recommend the collection maintainers deprecate a feature in one Ansible major version and do not remove that feature for one year, or at least until the next major Ansible version. For example, deprecate the feature in 3.1.0, and do not remove the feature until 5.0.0, or 4.0.0 at the earliest. Collections should use semantic versioning, such that the major collection version cannot be changed within an Ansible major version. Thus the removal should not happen before the next major Ansible community package release. This is up to each collection maintainer and cannot be guaranteed.
 
 ansible-core deprecation cycle
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The cycle is normally across 4 feature releases (2.x.y, where the x marks a feature release and the y a bugfix release),
-so the feature is normally removed in the 4th release after we announce the deprecation.
-For example, something deprecated in 2.9 will be removed in 2.13, assuming we don't jump to 3.x before that point.
-The tracking is tied to the number of releases, not the release numbering.
+The cycle is normally across 4 feature releases (2.x.y, where the x marks a feature release and the y a bugfix release), so the feature is normally removed in the 4th release after we announce the deprecation. For example, something deprecated in 2.9 will be removed in 2.13, assuming we don't jump to 3.x before that point. The tracking is tied to the number of releases, not the release numbering.
```

Comment:
```suggestion
The cycle is normally across 4 feature releases (2.x.y, where the x marks a feature release and the y a bugfix release), so the feature is normally removed in the 4th release after we announce the deprecation. For example, something deprecated in 2.9 will be removed in 2.13, assuming we do not jump to 3.x before that point. The tracking is tied to the number of releases, not the release numbering.
```
just to follow the "best practices" guide

Reply:
ah, thanks, I missed that one